static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.ts

```diff
@@ -148,6 +155,14 @@ declare global {
 				bidders: BidderCode[];
 				config: {
 					customPriceBucket?: PrebidPriceGranularity;
+					/**
+					 * This is a custom property that has been added to select
+					 * a price bucket based on the width and height of the slot
```

Comment:
Minor if you think it is accurate and helps!

```suggestion
					 * This is a custom property that has been added to our fork of prebid.js
					 * to select a price bucket based on the width and height of the slot
```